drivers/adc/ad7124/ad7124.h

```diff
@@ -247,6 +250,9 @@
 #define AD7124_CFG_REG_REF_SEL(x)  ((x) & 0x3) << 3
 #define AD7124_CFG_REG_PGA(x)      (((x) & 0x7) << 0)
 
+/* PGA Gain Value */
+#define AD7124_PGA_GAIN(x)     (1 << (x))
```

Comment:
again not related to this commit.

Reply:
will split this and related #define changes out into separate pull request